src/main/java/org/gaul/s3proxy/S3ProxyHandler.java

```diff
@@ -140,7 +140,14 @@
     private static final Set<String> SIGNED_SUBRESOURCES = ImmutableSet.of(
             "acl", "delete", "lifecycle", "location", "logging", "notification",
             "partNumber", "policy", "requestPayment", "torrent", "uploadId",
-            "uploads", "versionId", "versioning", "versions", "website"
+            "uploads", "versionId", "versioning", "versions", "website",
+            // Added from com.amazonaws.services.s3.internal.RestUtils
+            "tagging", "cors", "restore", "replication", "accelerate",
+            "inventory", "analytics", "metrics",
+            // Override-response-header parameters aslo need be signed
+            "response-content-type", "response-content-language",
+            "response-expires", "response-cache-control",
+            "response-content-disposition", "response-content-encoding"
```

Comment:
Can you only include the `response-*` parameters since S3Proxy does not support ones like `cors` and similar.  Also please keep the alphabetization.